.github/workflows/ci.yml

```
@@ -173,15 +173,15 @@ jobs:
       - name: Checkout actions repo
         uses: actions/checkout@v3
         with:
-          ref: explorer
           token: ${{ secrets.DEVOPS_ACTIONS_REPO_PAT }}
           repository: ${{ secrets.DEVOPS_ACTIONS_REPO }}
 
       - name: Deploy Explorer
         uses: ./actions/deploy
         with:
-          docker_tag: ${{ needs.build-publish.outputs.docker_image_digest }}
-          application_path: manifests/sites/explorer/${{ env.DEPLOY_ENV }}/base/kustomization.yaml
+          docker_image: hirosystems/explorer
```

Nit: we can generalize this part like so:
```suggestion
          docker_image: ${{ github.repository }}
```